air_windows.toml

```diff
@@ -0,0 +1,46 @@
+root = "."
```

Comment:
Let's rename this file to `.air_windows.toml` - Keeps it hidden for the linux guys

Reply:
You renamed the file and pushed the new name, but this is still here! You can push the "removed" file from git, it should appear as `deleted` when you do a `git status` - just do `git add .ait_windows.toml` after the deletion, and the commit it after that.